### PR TITLE
chore: create sync history table

### PR DESCRIPTION
**Pull request by lucasmarshall**

### CODE REVIEW

File: packages/db/prisma/schema.prisma [hunk 1]
```diff
@@ -166,3 +167,18 @@ model CrmOpportunity {
   @@unique([connectionId, remoteId])
   @@map("crm_opportunities")
 }
+
+model SyncHistory {
+  id             String     @id @default(uuid())
+  // contact, lead, account, opportunity, etc.
+  object         String
```
Comment (albertyfwu) [inline packages/db/prisma/schema.prisma:174]: we should standardize on `object` or `model`. i think we're using both in the code...

File: packages/db/prisma/schema.prisma [hunk 2]
```diff
@@ -166,3 +167,18 @@ model CrmOpportunity {
...
+  // success | error
+  result         String
+  startTimestamp DateTime   @default(now()) @map("start_timestamp")
```
Comment (albertyfwu) [inline packages/db/prisma/schema.prisma:177]: why do we want default here?

File: packages/db/prisma/schema.prisma [hunk 3]
```diff
@@ -33,19 +33,20 @@ model Integration {
...
   credentials   Json // {type, access_token, refresh_token, expires_at, raw}
-  customer      Customer    @relation(fields: [customerId], references: [id], onDelete: Cascade)
-  customerId    String      @map("customer_id")
-  createdAt     DateTime    @default(now()) @map("created_at")
-  updatedAt     DateTime    @updatedAt @map("updated_at")
+  customer      Customer      @relation(fields: [customerId], references: [id], onDelete: Cascade)
+  customerId    String        @map("customer_id")
+  createdAt     DateTime      @default(now()) @map("created_at")
+  updatedAt     DateTime      @updatedAt @map("updated_at")
+  SyncHistory   SyncHistory[]
```
Comment (albertyfwu) [inline packages/db/prisma/schema.prisma:49]: lowercase? 

also, do we need to call it `syncHistoryList`? For connections, we're calling it `connections: Connection[]`

Comment (lucasmarshall) [reply packages/db/prisma/schema.prisma:49]: Will update. This is the way the prisma extension for vscode auto-created this.

File: packages/db/prisma/schema.prisma (hunk 2 above)
Comment (lucasmarshall) [reply packages/db/prisma/schema.prisma:178]: I guess we dont

File: packages/db/prisma/schema.prisma (hunk 1 above)
Comment (lucasmarshall) [reply packages/db/prisma/schema.prisma:174]: will switch to model

**Review by albertyfwu — APPROVED** [review]

**Pull request merged**